pam-biometric.mk

```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+STRAPPROJECTS       += pam-biometric
+PAM-BIOMETRIC_VERSION := 1.0.0
+DEB_PAM-BIOMETRIC_V   ?= $(PAM-BIOMETRIC_VERSION)
+
+ifeq (,$(findstring darwin,$(MEMO_TARGET)))
+
+pam-biometric-setup: setup
+	-wget -q -nc -O $(BUILD_SOURCE)/pam-biometric-$(PAM-BIOMETRIC_VERSION).tar.gz https://github.com/sunflsks/pam-biometric/archive/refs/tags/$(PAM-BIOMETRIC_VERSION).tar.gz
+	$(call EXTRACT_TAR,pam-biometric-$(PAM-BIOMETRIC_VERSION).tar.gz,pam-biometric-$(PAM-BIOMETRIC_VERSION),pam-biometric)
+	mkdir -p $(BUILD_STAGE)/pam-biometric/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/pam
+	
+ifneq ($(wildcard $(BUILD_WORK)/pam-biometric/.build_complete),)
+pam-biometric:
+	@echo "Using previously built pam-biometric."
+else
+pam-biometric: pam-biometric-setup openpam
+	cd $(BUILD_WORK)/pam-biometric && \
+	$(CC) -shared pam-biometric.m $(CCFLAGS) -lpam -framework Foundation -framework LocalAuthentication -o pam-biometric.so && \
+	$(GINSTALL) -Dm755 pam-biometric.so $(BUILD_STAGE)/pam-biometric/$(MEMO_PREFIX)/$(MEMO_SUB_PREFIX)/lib/pam
```

Comment:
remove the `/` in SUB/PREFIX

Reply:
I don't see SUB/PREFIX anywhere?

Reply:
`$(MEMO_PREFIX)/$(MEMO_SUB_PREFIX)` should be `$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)`

Reply:
Oh, gotcha. Fixing that now.